sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/models/ServiceBusDescribedType.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.models;
+
+import org.apache.qpid.proton.amqp.DescribedType;
+
+/**
+ * Use described type to send OffsetDatetime, Duration, URI etc. on the wire.
+ */
+public class ServiceBusDescribedType implements DescribedType {
+
+    private final Object descriptor;
```

Comment:
Should it be type `Symbol`?

Reply:
The case is that if you only support `Symbol`, make your ctor only take `Symbol`. So no one will accidentally put a wrong type into this class.

You can certainly have another ctor take `Numeric` is needed later.

Not a big issue, as the class is now in implementation.